packages/ecs-gpu-init/ecs-gpu-init.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Initialize ECS agent GPUs config
+Documentation=https://aws.amazon.com/documentation/ecs/
+Before=ecs.service
+After=configured.target
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/ecs-gpu-init
+
+[Install]
+WantedBy=multi-user.target
+RequiredBy=ecs.service
```

Comment:
Is there any reason not to run this in `preconfigured.target` with the usual safeguards for oneshots?